routes/truckRoutes.js

```diff
@@ -1 +1,28 @@
-// put your Express router code in here
\ No newline at end of file
+var express = require('express');
+var trucks = require('../trucks');
+var bodyParser = require('body-parser');
+var router = express.Router();
+var urlEncoded = bodyParser.urlencoded({ extended: false });
+
+router.route('/')
+  .get(function (request, response){
+    var truckList = trucks.getTrucks();
+  
+    response.send(truckList);
+})
+.post(urlEncoded, function(request, response){
+  var newTruck = trucks.addTruck(truck); 
```

Comment:
You're passing in truck, but truck is undefined at this point. You have to assign it something first, or you need to pass in the request.body directly.